frontend/querybuilder/watcher.ts

```diff
@@ -5,7 +5,6 @@ import chokidar from 'chokidar';
 import chalk from 'chalk';
 import { main, ToolError, MainOptions, SCHEMA_PATH, LIB_PATH_PARTS, DOT_GRAPHQL } from './querybuilder';
 import { debouncer } from './util';
-import autobind from 'autobind-decorator';
```

Comment:
It's easier to just not use autobind for now, rather than installing whatever babel plugin is required for using decorators.